libs/island-ui/core/src/lib/Breadcrumbs/NewBreadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC, ReactNode } from 'react'
+import { Text } from '../Text/Text'
+import { Tag } from '../Tag/Tag'
+import { TagVariant } from '../Tag/Tag'
+import { Colors } from '@island.is/island-ui/theme'
+import cn from 'classnames'
+import * as styles from './Breadcrumbs.treat'
+
+export interface BreadCrumbItem {
+  title: string
+  href?: string
+  slug?: string[]
+  typename?: string
+  isTag?: boolean
+}
+
+interface BreadcrumbsProps {
+  items: BreadCrumbItem[]
+  label?: string
+  color?: keyof typeof styles.breadcrumb
+  tagVariant?: TagVariant
+  separatorColor?: Colors
+  renderLink?: (link: ReactNode, item: BreadCrumbItem) => ReactNode
+}
+
+export const NewBreadcrumbs: FC<BreadcrumbsProps> = ({
```

Comment:
kinda weird naming a component "New"

Reply:
lol this was a wip, it's now just "breadcrumbs" forgot to delete this file 🤦